lib/gis/parser_standard_options.c

```diff
@@ -786,6 +787,16 @@ struct Option *G_define_standard_option(int opt)
         Opt->description = _("Number of threads for parallel computing");
         break;
 
+    case G_OPT_M_SEED:
+        Opt->key = "seed";
+        Opt->type = TYPE_INTEGER;
+        Opt->required = NO;
+        Opt->label = _("Seed for random number generator");
+        Opt->description =
+            _("The same seed can be used to obtain same results"
+              " or random seed can be generated by other means.");
```

Comment:
This leave out the allusion to generating seeds externally (which would need more explanation anyway) and adds more details on what happens without the explicit seed (assuming the auto-generated seed behavior as an alternative).

```suggestion
            _("Using the same seed ensures identical results, "
              "while a randomly generated seed produces different outcomes "
              "in each run.");
```